### PR TITLE
New Comments Test

### PR DESCRIPTION
**Pull request by SamarthFlam**
Test pull request

### CODE REVIEW

File: code-review/ViewController.swift [hunk 1]
```diff
@@ -27,6 +27,8 @@ import UIKit
 
 // these are all new changes for PR
```
Comment (shivflam) [inline code-review/ViewController.swift:29]: change code

**Review by shivflam — COMMENTED** [review]
do check this comment

File: code-review/ViewController.swift (hunk 1 above)
Comment (SamarthFlam) [reply code-review/ViewController.swift:29]: Ill do the changes

**Review by SamarthFlam — COMMENTED** [review]
Test this

**Review by shivflam — DISMISSED** [review]
do correct the directory structure

**Review by shivflam — APPROVED** [review]
approved

**Comment by SamarthFlam** [comment]
Every thing approved

**Pull request merged**